examples/deployment/kubernetes/deploy.sh

```diff
@@ -3,50 +3,72 @@
 # - Cluster has already been created & configured using the create.sh script
 # - Go 1.9 is installed
 
+function checkEnv() {
+  if [ -z ${PROJECT_NAME+x} ] ||
+     [ -z ${CLUSTER_NAME+x} ] ||
+     [ -z ${MASTER_ZONE+x} ] ||
+     [ -z ${CONFIG_MAP+x} ]; then
+    echo "You must either pass an argument which is a config file, or set all the required environment variables"
```

Comment:
nit: Add `>&2` to send this error message to stderr.

Reply:
done